src/tests/class_tests/openms/CMakeLists.txt

```diff
@@ -81,6 +81,12 @@ endif()
 foreach(_class_test ${TEST_executables})
   add_executable(${_class_test} source/${_class_test}.cpp)
   target_link_libraries(${_class_test} ${OpenMS_LIBRARIES})
+
+  # Suppress std::move warning for MSSpectrum_test and MSChromatogram_test
+  if (NOT MSVC AND (_class_test STREQUAL "MSSpectrum_test" OR _class_test STREQUAL "MSChromatogram_test"))
+    target_compile_options(${_class_test} PUBLIC -Wno-pessimizing-move)
```

Comment:
sorry I'm a bit late to the party, but why not disable the warning in the test itself? e.g. using
`#pragma clang diagnostic ignored "-Wpadded"`
This way, it is easier to remember that the warning is disabled there (knowing this section of the build system seems only possible if your name is JP :))

Reply:
Yep I understand such way of suppressing warning and agree with your points. But I know that the `#pragma`  syntax for disabling warning is sorta difference for MSVC, so I'm not sure whether using `#pragma clang diagnostic ignored` will break Windows CI again...

Reply:
Maybe I can try building on a Windows machine using GitHub Actions to test the effect.

Reply:
check out these lines https://github.com/OpenMS/OpenMS/blob/develop/src/openms/source/SYSTEM/SysInfo.cpp#L38 you can also put the pragma in such blocks if you want to only enable them on linux or mac

Reply:
Thanks for the suggestions! I've updated my code (and rebased to `develop`)